CodeForFoco/volunteercore
# User Story: I can modify a user's admin status

To Test this:

1. Login as an admin
2. Go to "Users"
3. Click "Edit" on a user
4. use the admin checkbox to add/remove admin status
5. Submit
6. Go back to "users"
7. You'll see "Admin | _username_" on a user you added admin status to
